Updated aws-cluster-autoscaler Instructions

pluttrell opened the pull request:
The aws-cluster-autoscaler instructions were revamped to describe the
required configuration parameters, how to supply them, and an example.

Also added instructions on how to verify the required parameters are
successfully configured.

pluttrell commented:
I signed it!

lachie83 commented:
@mgoodness can you chime in on this one?

prydonius commented:
@mgoodness gentle ping

mikesplain commented:
@pluttrell you may want to add something in the docs to mention that version numbers of the docker image very much correlate to your kubernetes cluster version (https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#releases).  I was going to add it to a PR but saw all your great work here.

pluttrell committed:
Updated aws-cluster-autoscaler Instructions

The aws-cluster-autoscaler instructions were revamped to describe the
required configuration parameters, how to supply them, and an example.

Also added instructions on how to verify the required parameters are
successfully configured.

mgoodness committed:
README tweaks

mgoodness commented:
@mikesplain Let's make that change/addition in your PR instead. Merging this as-is.

mgoodness commented:
@k8s-bot ok to test

mgoodness commented:
/lgtm
